provider/cmd/app/app.go

```diff
@@ -80,6 +81,10 @@ type LibvirtOptions struct {
 	PreferredMachineTypes []string
 
 	Qcow2Type string
+
+	DisableNuma      bool
```

Comment:
are their libvirt options or libvirt-provider options?

Reply:
I believe this section of libvirt-provider has options to interact with libvirt.

Reply:
yes but we interact with libvirt on multiple levels. Primary we have libvirt client and settings specific for libvirt daemon. After we have this tuna functions which modified libvirt domain but they are independent on client or daemon. If you look on fields, you have multiple different fields in one structure which serve for different purpose and they are use in different places of code. So i prefer create for tunning separate structure. It will improve readability of code and you will very nice see which fields are specific for tunning only.

Reply:
Would you like to suggest any name for the structure.

Reply:
Tunning, Tune, ... up to you